llm/llama-4/llama4.yaml

```diff
@@ -0,0 +1,39 @@
+envs:
+  MODEL_NAME: meta-llama/Llama-4-Maverick-17B-128E-Instruct-FP8
+  # MODEL_NAME: meta-llama/Llama-3.2-3B-Vision
+  HF_TOKEN: # TODO: Fill with your own huggingface token, or use --env to pass.
+
+resources:
+  accelerators: { H100:8, H100:12, H200:8, GH200:8, B100:8, B200:8, GB200:8 }
+  cpus: 32+
+  disk_size: 512  # Ensure model checkpoints can fit.
+  disk_tier: best
+  ports: 8081
+
+setup: |
+  # uv pip install vllm>0.8.2
```

Comment:
Looks like https://github.com/vllm-project/vllm/pull/16104 is merged to 0.8.3, hopefully should be out in the release soon! 🎉